src/main/java/org/humancellatlas/ingest/submission/web/SubmissionController.java

```diff
@@ -365,7 +365,7 @@
 
     @GetMapping("/submissionEnvelopes/{id}" + Links.SUBMISSION_CONTENT_LAST_UPDATED_URL)
     ResponseEntity<?> getContentLastUpdated(@PathVariable("id") SubmissionEnvelope submissionEnvelope) {
-        Instant lastUpdateDate = submissionEnvelopeService.getSubmissionContentLastUpdated(submissionEnvelope);
-        return ResponseEntity.ok(lastUpdateDate.toString());
+        Optional<Instant> lastUpdateDate = submissionEnvelopeService.getSubmissionContentLastUpdated(submissionEnvelope);
+        return ResponseEntity.ok(lastUpdateDate.isPresent() ? lastUpdateDate.get().toString() : null);
```

Comment:
What does the null case mean here?
Would it be correct to have an `ok(null)` return value?